openpype/tools/project_manager/project_manager/window.py

```diff
@@ -82,11 +89,23 @@ def __init__(self, parent=None):
         add_asset_btn.setObjectName("IconBtn")
         add_task_btn.setObjectName("IconBtn")
 
+        add_misc_folders_label = QtWidgets.QLabel(
```

Comment:
I would move it to a right side of row where `Create Project` button is and without label.

Reply:
Also rename it to `create_folders_btn`...

Reply:
done